src/csync/csync_private.h

```diff
@@ -76,8 +77,8 @@ struct OCSYNC_EXPORT csync_s {
       void *update_callback_userdata = nullptr;
 
       /* hooks for checking the white list (uses the update_callback_userdata) */
-      int (*checkSelectiveSyncBlackListHook)(void*, const char*) = nullptr;
-      int (*checkSelectiveSyncNewFolderHook)(void*, const char* /* path */, const char* /* remotePerm */) = nullptr;
+      int (*checkSelectiveSyncBlackListHook)(void*, const QByteArray &) = nullptr;
```

Comment:
Slightly, it goes with the change of the return value of `csync_rename_adjust_path_source` to QByteArray, ultimately because of the folder_renamed_to/from data structure changes.